.github/mergify.yml

```diff
@@ -52,7 +52,15 @@ pull_request_rules:
         - label=merge+no rebase
       - '#approved-reviews-by>=2'
       - '#changes-requested-reviews-by=0'
-      - '#check-failure=0'
+      # oy
```

Comment:
oy?

Reply:
Comment on needing to copy those from branch protection. Those weren't supposed to have been removed; it looks like I may have missed a rebase somewhere in the original PR.